src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -238,7 +238,7 @@ public PurchaseOrder CreateOrder(PurchaseOrder order, IEnumerable<string> privil
                 this.SetDetailFieldsForCreation(detail, newOrderNumber);
 
                 this.PerformDetailCalculations(
```

Comment:
I think I would have just checked if order.DocumentType was a returns or credit here, and not called this PerformDetailCalculations at all in that case.. 

I think thats a lot clearer and also maybe better? since presumably we don't want anything about the order value to be changed for returns/credit orders anway? E.g. we don't want them changing the qty either as that will change the value too?

Reply:
@richardiphillips is that right? Should we be stopping the values of these orders changing at all (by restricting changes to price, qty etc completely)

Or just stopping the unit price change?

Reply:
i think you can change the qty. might only be returning a few broken ones